renderlib/ViewerWindow.cpp

```diff
@@ -50,6 +51,21 @@ ViewerWindow::setSize(int width, int height)
   // TODO do whatever resizing now?
 }
 
+void
+ViewerWindow::showAreaLightGizmo(bool show)
+{
+  // check if m_areaLightTool is in m_tools
+  bool isShowing = std::find(m_tools.begin(), m_tools.end(), m_areaLightTool) != m_tools.end();
+  if (isShowing == show) {
+    return;
+  }
+  if (show) {
+    m_tools.push_back(m_areaLightTool);
+  } else {
+    m_tools.erase(std::remove(m_tools.begin(), m_tools.end(), m_areaLightTool), m_tools.end());
```

Comment:
If there are other tools in `m_tools` after `m_areaLightTool`, would this potentially erase/hide other tools unintentionally?

Reply:
This is sort of a standard recipe for "get rid of one element from a vector, by value".  std::remove finds the thing and makes sure it is shifted to the end of the list, and returns an iterator to the start of the removed things... and vector::erase drops it from m_tools.